src/main/java/com/emc/ecs/servicebroker/service/EcsService.java

```diff
@@ -704,6 +732,10 @@ protected void prepareBucketWipe() throws URISyntaxException {
         bucketWipe = bucketWipeFactory.getBucketWipe(broker);
     }
 
+    private void prepareS3Client() {
```

Comment:
remove in favor of autowiring

Reply:
check [this comment](https://github.com/EMCECS/ecs-cf-service-broker/pull/5#discussion_r1805151870)